packages/documentation-site/patternfly-docs/content/design-guidelines/content/capitalization.md

```diff
@@ -7,70 +7,75 @@ Consistent capitalization adds clarity and creates unity across your products' U
 
 ## Capitalization types
 
+We recommend using title and sentence case as appropriate:
 
-**Title case** capitalizes only the first letter of each word in a sentence except for smaller words like articles and conjunctions that have fewer than four letters.
+- **Title case** capitalizes only the first letter of each word in a sentence, except for smaller words with fewer than four letters, like articles and conjunctions. 
 
-Example: "PatternFly Design Resources With Tips and Best Practices"
+    - For example: "PatternFly Design Resources with Tips and Best Practices."
 
-Hyperlinked articles in a UI are not displayed as titles. Instead, the copy typically reads something like, "Learn more about [tool name]." 
+    - **Note:** Hyperlinked articles in a UI are not displayed as titles. Instead, the copy typically reads something like, "Learn more about [tool name]." 
 
-**Sentence case** only capitalizes the first letter of the first word in a title, heading, subtitle, or subheading except for proper nouns, product names, acronyms, and initialisms.
+- **Sentence case** only capitalizes the first letter of the first word in a title, heading, subtitle, or subheading except for proper nouns, product names, acronyms, and initialisms. 
+    
+    - For example: “PatternFly’s design resources with tips and best practices.”
 
-Example: “PatternFly’s design resources with tips and best practices” 
+## Capitalization in breadcrumb trails
```

Comment:
I feel like this section should come at the end, because right now, if I just read the "Capitalization types" I would be left wondering which one I should be using when, and it's not immediately apparent that its addressed later down the page. So I would personally put this section after capitalization across PF and across Red Hat UIs!